docker-compose.yml

```diff
@@ -33,7 +33,7 @@ services:
       timeout: 5s
       retries: 18
   node_2:
-    image: scylladb/scylla-nightly
+    image: scylladb/scylla-nightly:6.0.0-rc2-0.20240602.cbf47319c1f7
```

Comment:
@roydahan , @sylwiaszunejko , WDYT does it work to have somethign like `SCYLLA_TABLET_IMAGE`, instead of hardcoded one?

Only problem I see here is that driver-matrix test can use this either `docker-compose.yaml` or `integration.sh`

Reply:
I am not sure, when we have 6.0 we would not need to have different scylla version for tablet/non-tablet tests and then there will be no need for `SCYLLA_TABLET_IMAGE`, we will just  bump `SCYLLA_IMAGE` to `scylladb/scylla:6.0.0` and use it for all of the nodes

Reply:
Agreed